lib/archive/stripper_test.go

```diff
@@ -132,17 +132,19 @@ func TestSingleStripper(t *testing.T) {
 		}
 
 		// make the buf just that little bit bigger to allow for errrors in the copy if they would occur
-		buf := make([]byte, size+10)
+		buf := make([]byte, size)
 
-		fmt.Printf("Reading data for file %d\n", i)
+		t.Logf("Reading data for file %d\n", i)
 		n, err := tr.Read(buf)
 
-		if !assert.NoError(t, err, "No expected errors from file data copy") {
+		if !assert.Equal(t, n, size, "Expected file data size to match target generated size") {
```

Comment:
There is a `require` that can be used instead of `assert` now, which does `FailNow` if the assertion fails. Simplifies the code significantly.